src/main.ts

```diff
@@ -85,7 +86,7 @@ async function handler(req: Request): Promise<Response> {
           return await storage.deleteSession(id, key);
       }
     }
-
+    // fallthrough
     case "sessions": {
```

Comment:
Oh shit, this was introduced in #4. Can you fix it by adding a default case instead of the comment?

Reply:
> fix it by adding a default case

There is one. in the outer switch, I can add a more specific one though.

Reply:
Seems reasonable to do, yes